### DISCUSSION TITLE
GITHUB Copilot

### DISCUSSION THREAD

**Discussion by ghost** [community/community#49670] [Copilot Conversations]
### Select Topic Area

General

### Body

I've joined the gitbhub global campus, it says that github copilot is free for these members of this program, where can i use the copilot now for free?

**Comment by Ntshangase** [comment]
Hello, I'm also a global campus student, I use github copilot on VS Code and the setup wasn't that hard.

**Reply by ghost** [reply]
Same, it's really cool. I figured it out.

**Reply by ghost** [reply]
How can I add friends on github, any idea?

**Reply by rayhanalshorif133** [reply]
@thecarwalks  see this documentation 
https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-access-to-your-personal-repositories/inviting-collaborators-to-a-personal-repository

**Reply by Ntshangase** [reply]
I'm not sure with what. you mean when you say "add friends" but I know that you can follow other peoples profile if you find them interesting.

**Comment by rayhanalshorif133** [comment]
github copilot on VS Code, using this extentaion : 
 
Name: GitHub Copilot
Id: GitHub.copilot
Description: Your AI pair programmer
Version: 1.77.9225
Publisher: GitHub
VS Marketplace Link: https://marketplace.visualstudio.com/items?itemName=GitHub.copilot
